### PR TITLE
Update domain lisk.io to lisk.com

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

- The lisk domain was changed from lisk.io to lisk.com, which was not reflected in codebase

### How was it solved?

- Updated all the domain references

### How was it tested?

- Click on new domain links and check if they navigate to correct page

### CODE REVIEW

File: elements/lisk-client/README.md [hunk 1]
```diff
@@ -76,4 +76,4 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 [lisk core github]: https://github.com/LiskHQ/lisk
-[lisk documentation site]: https://lisk.io/documentation/lisk-elements
+[lisk documentation site]: https://lisk.com/documentation/lisk-sdk/references/lisk-elements/index.html
```
Comment (shuse2) [inline elements/lisk-client/README.md:79]: I think we dont need `index.html`? same for other places

**Review by shuse2 — COMMENTED** [review]
looks good in general

**Review by shuse2 — APPROVED** [review]

File: sdk/README.md [hunk 2]
```diff
@@ -20,19 +20,19 @@ We strictly discourage anyone from using the beta release of the Lisk SDK for an
 
 The only application built using the Lisk SDK currently feasible for production usage is [Lisk Core](https://github.com/liskhq/lisk-core), the client of the Lisk network itself.
 
-Please be advised, although we have stabilised the architecture of SDK, we cannot guarantee blockchains created with the beta release of the Lisk SDK will remain compatible with our planned release candidates.
+Please be advised, although we have stabilized the architecture of SDK, we cannot guarantee blockchain created with the beta release of the Lisk SDK will remain compatible with our planned release candidates.
```
Comment (Incede) [inline sdk/README.md:23]:
```suggestion
Please be advised, although we have stabilized the architecture of SDK, we cannot guarantee a blockchain created with the beta release of the Lisk SDK will remain compatible with our planned release candidates.
```

File: elements/README.md [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
 ## What is Lisk Elements
 
-Lisk Elements is a collection of libraries, each of them implementing some form of blockchain application functionality such as cryptography, transactions, p2p, etc. Each library is designed to be compatible with the [Lisk Protocol](https://lisk.io/documentation/lisk-protocol).
+Lisk Elements is a collection of libraries, each of them implementing some form of blockchain application functionality such as cryptography, transactions, p2p, etc. Each library is designed to be compatible with the [Lisk Protocol](https://research.lisk.com/).
```
Comment (Incede) [inline elements/README.md:9]: Any reason for changing this to `research.lisk.com` ?
I don't see anything that goes by `Lisk Protocol` here.

How about `https://lisk.com/documentation/lisk-sdk/protocol/index.html`

File: elements/lisk-api-client/README.md [hunk 4]
```diff
@@ -25,4 +25,4 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 [lisk core github]: https://github.com/LiskHQ/lisk
-[lisk documentation site]: https://lisk.io/documentation/lisk-elements
+[lisk documentation site]: https://lisk.com/documentation/lisk-sdk/references/lisk-elements
```
Comment (Incede) [inline elements/lisk-api-client/README.md:28]: Wouldn't it be better if we link to the exact sub-section ?
Same for all other readme's.
```suggestion
[lisk documentation site]: https://lisk.com/documentation/lisk-sdk/references/lisk-elements/api-client.html
```

File: framework-plugins/lisk-framework-dashboard-plugin/README.md [hunk 5]
```diff
@@ -35,4 +35,4 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 [lisk core github]: https://github.com/LiskHQ/lisk
-[lisk documentation site]: https://lisk.io/documentation/lisk-elements
+[lisk documentation site]: https://lisk.com/documentation/lisk-sdk/references/lisk-elements
```
Comment (Incede) [inline framework-plugins/lisk-framework-dashboard-plugin/README.md:38]: Why are we linking to `elements` docs for `framework-plugins` ?

**Review by Incede — CHANGES_REQUESTED** [review]

**Comment by ManuGowda** [comment]
Good catch @Incede

**Review by Incede — APPROVED** [review]

**Pull request merged**